tests/integration/requirements.txt

```diff
@@ -1,5 +1,8 @@
 juju
 pytest
+Pillow
```

Comment:
Is this needed? I couldn't see it get imported anywhere? Maybe it is the `PIL` package?

Reply:
It's used in integration tests (tests/integration/test_core.py)